src/packager.ts

```diff
@@ -320,6 +320,11 @@ export async function checkWineVersion(checkPromise: Promise<string>) {
     wineVersion = wineVersion.substring(0, spaceIndex)
   }
 
+  const rcIndex = wineVersion.indexOf("-rc")
```

Comment:
Then it may be simply makes sense to replace all "-" on "."? In this case, you will see the difference between wine-2.0-rc1 and wine-2.0-rc2. They will be given to the form and 2.0.rc1 2.0.rc1. Semver be able to recognize this type of version?